cmake/TargetDoxygenDox.cmake

```diff
@@ -66,7 +66,7 @@ IF (DOXYGEN_FOUND)
       STRING(REGEX REPLACE "[/]" "\\\\" HHP_FILE ${TMP} )
       # MESSAGE(SEND_ERROR "DBG  HHP_FILE=${HHP_FILE}")
       ADD_CUSTOM_TARGET(winhelp ${HTML_HELP_COMPILER} ${HHP_FILE})
-      ADD_DEPENDENCIES (winhelp doc)
+      ADD_DEPENDENCIES (winhelp dox)
```

Comment:
Same comment as above.. what was the problem ?

Reply:
the problem here is that we create a dependence named "dox" line 56, and here we want to attach "doc", which is not referenced in this file.